templates/.circleci/config.yml

```diff
@@ -5,7 +5,9 @@ jobs:
       - image: circleci/ruby:__ruby_version__-stretch-node-browsers
         environment:
           RAILS_ENV: test
-      - image: circleci/postgres:9.6.5
+      - image: circleci/postgres:13.0
+          environment:
+            POSTGRES_HOST_AUTH_METHOD:trust
```

Comment:
The step that tests the generated rails app is still using [9.6.5](https://github.com/TheGnarCo/gnarails/blob/d2209ae8dd963ee539f497dbd32798bad5d92154/.circleci/config.yml#L73), which you can also see in in the CI [output](https://app.circleci.com/pipelines/github/TheGnarCo/gnarails/2/workflows/9887e7bb-2f4f-4cc3-989f-66276c4f1210/jobs/933).

Now, ideally, the test-rails-app step of our CI process would use the generated CI config to test itself against, but I'll be honest - I'm not sure how to make that happen. 

At the very least, we should incorporate this change into that place as well.

Reply:
Oh yeah, good catch! 49d3c636f830f9ed2f4eb9badca8a55d4fed59a9